experimental/saiextensions.h

```diff
@@ -37,7 +37,7 @@
 /**
  * @brief Extensions to SAI APIs
  *
- * @flags
+ * @flags free
```

Comment:
what does "free" mean here and why this is needed?

Reply:
this is part of metadata, there are couple flags types, like strict, free, mixed etc, for more info analyze this PR https://github.com/opencomputeproject/SAI/pull/1314